ts/Extensions/Annotations/Popup.ts

```diff
@@ -826,20 +826,25 @@ H.Popup.prototype = {
                     storage.reverse(); // (#14691)
                 }
 
-                storage.forEach(function (genInput): void {
+                storage.forEach(function (genInput: any): void {
```

Comment:
Do we need this `any?

I noticed that the `getInput` is an array of objects with `type` and `value`.

Ref: https://github.com/highcharts/highcharts/commit/cc40c239b3f06baae0ed1167c808e178b239be5f#diff-276aaccade8e98f0f52f9dae7d9e60c247861424ae8a2cbbb84d8ceb1724393eR907

Reply:
This casting to any was written whenever the `genInput` was used that is why I decided to simplify it a bit and I moved that `as any` here.
I guess there might be a way to get rid of that nasty casting however this isn't the purpose of that PR.

Reply:
When it comes to `if (genInput[0] === true) {` it has to be like that.. 
Without that the popup in Annotation is failing